src/DssSpell.t.sol

```diff
@@ -2494,15 +2252,53 @@ contract DssSpellTest is DSTest, DSMath {
         // Hacking nxt price to 0x123 (and making it valid)
         bytes32 hackedValue = 0x0000000000000000000000000000000100000000000000000000000000000123;
 
-        ClipAbstract clipLINKA = ClipAbstract(addr.addr("MCD_CLIP_LINK_A"));
+        ClipAbstract clipYFIA = ClipAbstract(addr.addr("MCD_CLIP_YFI_A"));
 
-        hevm.store(address(addr.addr("PIP_LINK")), bytes32(uint256(4)), hackedValue);
+        hevm.store(address(addr.addr("PIP_YFI")), bytes32(uint256(4)), hackedValue);
 
-        assertEq(clipMom.tolerance(address(clipLINKA)), (RAY / 2)); // (RAY / 2) for 50%
+        assertEq(clipMom.tolerance(address(clipYFIA)), (RAY / 2)); // (RAY / 2) for 50%
 
         // Price is hacked, anyone can trip the breaker
-        clipMom.tripBreaker(address(clipLINKA));
+        clipMom.tripBreaker(address(clipYFIA));
+
+        assertEq(clipYFIA.stopped(), 2);
+    }
+
+    function testLerp() public {
+        LerpFabLike factory = LerpFabLike(addr.addr("LERP_FAB"));
+
+        assertEq(vow.hump(), 30 * MILLION * RAD);
+        assertEq(factory.count(), 0);
+
+        vote(address(spell));
+        scheduleWaitAndCast(address(spell));
+        assertTrue(spell.done());
+
+        assertEq(factory.count(), 1);
+        LerpLike lerp = LerpLike(factory.lerps("20210423_VOW_HUMP1"));
+
+        assertEq(vow.hump(), 30 * MILLION * RAD);
+
+        // Should do nothing as we are before the start date
+        lerp.tick();
+        assertEq(vow.hump(), 30 * MILLION * RAD);
+
+        // Should do nothing as we are exactly at the start date
+        lerp.tick();
+        assertEq(vow.hump(), 30 * MILLION * RAD);
+
+        hevm.warp(1619841600 + 1 days);
```

Comment:
Update to new timestamp requested in the spell.

Reply:
Done!